samples/custom_dispatch/vulkan/shaders/README.md

```diff
@@ -145,3 +145,22 @@ for instructions for CMake setup and building from source.
         --input=8xf32=4 \
         /tmp/example.vmfb
     ```
+
+## Udon Kernel Instructions
+
+(Pronounced U-DO-U, the second `u` is upside-down). This is a flow for authoring
```

Comment:
that is hilarious! Maybe we just say "udou" kernels.

Reply:
You're not really expressing your individuality if you keep all of the letters right side up.